.github/workflows/ci.yml

```diff
@@ -85,6 +85,42 @@ jobs:
       - name: Cleanup
         if: always()
         run: docker compose down --volumes
+
+  test-api:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ["3.11"]
+    steps:
+      - name: Checkout the repo
+        uses: actions/checkout@v3
+
+      - name: Python Setup
+        uses: actions/setup-python@v4
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      - name: Checkout Source
+        uses: actions/checkout@v3
+
+      - name: Install Dependencies
+        run: |
+          pip install --upgrade pip
+          pip install pytest-timeout
+          pip install pytest-cov
```

Comment:
I think it would be great to see this in the PR. Personally, I never look at the commit results.

Also, any idea why the links to the files don't work?

Reply:
It seems to link the files with a path to root folder so [without /api](https://github.com/EURODEO/e-soh/blob/ed77ae35c38c90c7444d05fb4c592ca2c18d59cf/main.py) instead of [with /api](https://github.com/EURODEO/e-soh/blob/ed77ae35c38c90c7444d05fb4c592ca2c18d59cf/api/main.py). I'll check how I can fix that and make it do the comment to pull request.